plugins/woocommerce/src/StoreApi/Routes/V1/CartApplyCoupon.php

```diff
@@ -67,6 +66,6 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 			throw new RouteException( $e->getErrorCode(), $e->getMessage(), $e->getCode() );
 		}
 
-		return rest_ensure_response( $this->schema->get_item_response( $cart ) );
+		return rest_ensure_response( $this->schema->get_item_response( $this->cart_controller->get_cart_instance() ) );
```

Comment:
I just wanted to learn why we are replacing the `$cart` variable defined in line 61. I felt that the previous code was easy to read. The new code,`return rest_ensure_response( $this->schema->get_item_response( $this->cart_controller->get_cart_instance() ) );`, seems a bit complicated to understand.

Reply:
We can leave the $cart variable for readability, I just noticed we were calling it too early as we might not need it if we throw the exception. It was just a minor nitpick, but I can instance the var just before using it if you prefer. 
The way it is done here saves the unnecessary memory allocation for a new var, but that is negligible 😉

Reply:
I'd prefer to have it as it was before for better readability of the code. Thank you so much for accepting the change. 🙂